<launch>
  <arg name="language" default="en"/>
  <arg name="intern_file" default="none"/> <!--$(find ontologenius)/file_intern/ontologenius.owl-->
  <arg name="robot_file" default="none"/>
  <arg name="human_file" default="none"/>
  <arg name="config_file" default="none"/>
  <arg name="display" default="false"/>
  <arg name="files" default=""/>


  <node name="ontologenius_core" pkg="ontologenius" type="ontologenius_multi" output="screen" args="-l $(arg language) -c $(arg config_file) -i $(arg intern_file) -d $(arg display) -r $(arg robot_file) -h $(arg human_file) $(arg files)"> </node>
  <node name="ontologenius_rest" pkg="ontologenius" type="REST.py" output="screen"> </node>
</launch>
